Formula/podman.rb

```diff
@@ -2,10 +2,12 @@ class Podman < Formula
   desc "Tool for managing OCI containers and pods"
   homepage "https://podman.io/"
   license "Apache-2.0"
+  revision 1
 
   stable do
     url "https://github.com/containers/podman/archive/v4.2.0.tar.gz"
     sha256 "15f8bc59025ccd97dc9212a552e7274dfb79e1633b02d6a2a7f63d747eadb2f4"
+    patch :DATA
```

Comment:
Is there a PR/commit for this patch? If yes, can we use the GitHub-hosted patch instead?

Reply:
The code in containers/common is vendored into containers/podman.

The PR in Podman that brought in this change is containers/podman#15437, but you'll see that it brings many unrelated changes and I'm not really sure whether it will even apply cleanly given the changes to files such as `go.mod` which are likely to depend on previous dependencies.

That's mostly why I preferred to craft a more contained patch only with the changes from the PR in containers/common that is relevant for this feature. Since the patch goes away with v4.3.0 of Podman (or whatever the next Podman release is), do you think it's unreasonable to keep it as a custom local patch until then?

Reply:
I'd rather wait for the next release then since it only impacts people with an unsupported configuration.

Reply:
Alright, so I found a way to apply the patches directly from GitHub, using the `directory` method inside a `patch do` block, so I updated the PR to do that instead.

If the main objection was the embedded patch, that should resolve it.

I do see a motivation to push this feature now (rather than wait for the next release of Podman) which is we won't forget about it when the next release is out (not even sure when that would be.)

But let me know if you'd still prefer to abandon this PR.